server/configuration.go

```diff
@@ -454,6 +470,12 @@ func (c *configuration) liveCaptionsEnabled() bool {
 	return false
 }
 
+func (p *Plugin) getAdminConfig() *configuration {
+	c := p.getConfiguration()
+	c.clientConfig = p.getClientConfig()
```

Comment:
This is easier for sure, but I wonder if it might cause confusion down the road? I've found that knowing the shape of `clientConfig` has helped keep track of what we send over and what we need to handle on the client (e.g. when double-checking that the shape of the callsConfig is correct). If we send over the entire `configuration` we'll now have to keep track of what part of that we actually use on the client -- we'll have to search through and try to reason about it (like I did right now to make sure we only need `TranscribeAPI` in the `callsConfig`). A getAdminConfig that specifies what we send over would be easier, imo.

Reply:
@cpoile I think admins should be allowed to read the whole thing since they can configure it in the first place so the shape would the full object, aka `configuration`. Creating another subset feels just more indirection to me, but I'm happy to reconsider.

Reply:
Certainly they should be able to see the configuration, but I'm thinking more about us trying to make sure we know what it is we're using from that. Maybe I'm just worried that at some point we'll use another field somewhere and forget to add it to the callsConfig because we don't need it stored in the state, and now we've got a hidden dependency. Just a thought, I'll let you decide what's best. :)

Reply:
@cpoile I reworked this a bit to keep it scoped to [what we added on the client side](https://github.com/mattermost/calls-common/commit/e463e78cc76302d1bdbd085d9e1846c456f48d45) as well. Let me know if that's closer to what you had in mind. See https://github.com/mattermost/mattermost-plugin-calls/pull/786/commits/d8fcf87f81f5acccdcb85d94ed63e9a939b53588